<launch>   
   <!-- lanzar nodo joy primero -->
    <arg name="dev" default="/dev/input/js0" />
    <node pkg="joy" type="joy_node" name="joy">
        <param name="dev" value="$(arg dev)" />
    </node>

    <!-- joy twist -->
    <node pkg="robocar" type="joy_teleop" name="twist">

        <param name="axis_linear" value="5" />
        <param name="axis_brake" value="2" />

        <param name="scale_linear" type="double" value="0.5" />
        <param name="offset_linear" type="double" value="0.5" />
        
        <param name="axis_angular" value="0" />
        <param name="scale_angular" type="double" value="0.34" />

        <param name="output_topic" type="string" value="/joy_cmd_vel">

    </node>
    <!-- deteccion de obstaculos -->
    <node pkg="robocar" type="obstacle_node.py" name="obstacle_node">

        <param name="range_topic" type="string" value="/laser">
        <!-- default values -->
    </node>

    <!-- red neuronal -->
    <node pkg="robocar" type="neural_node.py" name="neural_node">
        <!-- default values -->        
    </node>

    <!-- piloto -->
    <node pkg="robocar" type="pilot_node" name="pilot_node">

        <!-- default values --> 
    </node>


</launch>